index.src.html

```diff
@@ -3182,6 +3185,8 @@
   readonly attribute long long? timestamp;          // microseconds
   readonly attribute VideoColorSpace colorSpace;
 
+  VideoFrameMetadata getMetadata();
```

Comment:
Nit: consider just `metadata()` for consistency with `allocationSize()`.

Reply:
Let's go with metadata() for consistency.

Reply:
Done. It is a bit strange we do not have consistency across specs here.